test/unittests/tools/gnu/autotools_test.py

```diff
@@ -33,3 +35,28 @@ def test_source_folder_works(chdir_mock):
     autotools.autoreconf()
     chdir_mock.assert_called_with(autotools, os.path.normpath("/path/to/sources"))
     assert conanfile.command == 'autoreconf -bar foo'
+
+@pytest.mark.parametrize("install_strip", [False, True])
+def test_install_strip(install_strip):
+    """
+    When the configuration `tools.build:install_strip` is set to True,
+    the Autotools install command should invoke the `install-strip` target.
+    """
+
+    settings = MockSettings({"build_type": "Release",
+                             "compiler": "gcc",
+                             "compiler.version": "7",
+                             "os": "Linux",
+                             "arch": "x86_64"})
+    conanfile = ConanFileMock()
+    conanfile.settings = settings
+    conanfile.conf = Conf()
+    conanfile.conf["tools.build:install_strip"] = install_strip
```

Comment:
```suggestion
    conanfile.conf.define("tools.build:install_strip", install_strip)
```
The .conf is already defined by ``ConanFileMock``

Reply:
Thanks for the diff, done.